### PR TITLE
Seperate faq

### PR DESCRIPTION
**Pull request by Ciuffi**
I've moved the FAQ section to a top level section (under the livesite). I've also removed it from the individual hackathons. @panjenny0 will be adding the functionality to assign FAQ questions to certain hackathons. After jenny finishes her changes, we can make a new release with the FAQ and sponsor fixes.

(updated image since latest changes)
image

### CODE REVIEW

File: components/sidebar.js [hunk 1]
```diff
@@ -90,6 +91,42 @@ const Link = styled.a`
   }
 `;
 
+const MiddleLink = styled.a`
```
Comment (ianmah) [inline components/sidebar.js:94]: I'm thinking we can remove the `margin-top` from `Link`, and add a `margin-bottom` to `HeaderContainer` instead? Tbh I'm not sure if the Logout button needs a margin on top of it?

**Review by ianmah — COMMENTED** [review]
I was clicking around the pages and noticed there are some resizing issues when navigating using the sidebar.

I noticed pages `[id]` in the directory are correctly using the `<Page>` component, whereas the pages outside of that directory are not. I originally intended for all pages to use the `Page` component so that the container and sidebar are consistent across all pages. But, it looks like the `Page` component has been coupled with navbar - maybe we can modify `Page` to `includeNavbar` or not? (and then refactor all pages to use the Page component)

Edit: Made the refactor in #41

**Comment by Ciuffi** [comment]
I think thats a great idea, I'll adopt the page changes in this PR once your other PR is merged.

**Comment by Ciuffi** [comment]
@ianmah okay i've made the refactors and removed middleLink.

File: pages/faq.js [hunk 2]
```diff
@@ -236,7 +239,7 @@ export default function Faq({ currHackathon }) {
   }
 
   return (
-    <>
+    <Page hackathons={hackathons} currentPath="faq" navbarItems={null}>
```
Comment (ianmah) [inline pages/faq.js:242]:
```suggestion
    <Page hackathons={hackathons} currentPath="faq">
```
does this work?

**Review by ianmah — APPROVED** [review]
lgtm!

**Review by andykwan1124 — APPROVED** [review]
lgtm

**Pull request merged**